samples/scripts/externalclient-setup.js

```diff
@@ -84,8 +84,10 @@
 const fs = require('fs');
 const {promisify} = require('util');
 const {GoogleAuth} = require('google-auth-library');
+const {google} = require('googleapis');
 
 const readFile = promisify(fs.readFile);
+const iam = google.iam('v1');
```

Comment:
It's unfortunate that we would need to use the `googleapis` npm module here.  @alexander-fenster I don't believe we have a `@google-cloud/nodejs-iam` - but is there supposed to be a way to do this with a GAPIC?

Reply:
Hi, @JustinBeckwith, thanks for your feedback.
I am not sure I understand
> 'It's unfortunate that we would need to use the `googleapis` npm module here', 

I am importing the `googleapis` module and using IAM client library inside the module, which is following the requirement above. What are the restrictions here that the IAM client library isn't allowed in this repo, and what's the expectation for the solution?

Reply:
Hey @JustinBeckwith, unfortunately `@google-cloud` does not seem to contain the needed IAM APIs: https://github.com/googleapis/google-cloud-node#google-cloud-nodejs-client-libraries
There is `@google-cloud/iam-credentials` but this is for iamcredentials.

Currently, it seems only `googleapis` is documented for IAM: https://cloud.google.com/iam/docs/reference/libraries#iam-client-libraries-reference-nodejs

Reply:
Agreed.  We talked this through and I think we decided not to publish an IAM GAPIC for now, so this is fine.  Sorry for the confusion.